### PR TITLE
upscayl 2.9.9 (new cask)

### PR DESCRIPTION
**Pull request by aaronliu0130**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
  * Was refused due to 1. missing a signature 2. having an incorrect min version in the plist, both of which we should now have
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/u/upscayl.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+cask "upscayl" do
+  version "2.9.9"
+  sha256 "21cab689f557c5ddaee36ee9f6bf22bf042072cbb4e3c354e486e2ba2b86df3a"
+
+  url "https://github.com/upscayl/upscayl/releases/download/v#{version}/upscayl-#{version}-mac.dmg",
+      verified: "github.com/upscayl/upscayl/"
+  name "upscayl"
+  desc "Open Source AI Image Upscaler built with Linux-First philosophy"
```
Comment (bevanjkay) [inline Casks/u/upscayl.rb:8]:
```suggestion
  desc "AI image upscaler"
```

File: Casks/u/upscayl.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
+cask "upscayl" do
+  version "2.9.9"
+  sha256 "21cab689f557c5ddaee36ee9f6bf22bf042072cbb4e3c354e486e2ba2b86df3a"
+
+  url "https://github.com/upscayl/upscayl/releases/download/v#{version}/upscayl-#{version}-mac.dmg",
+      verified: "github.com/upscayl/upscayl/"
+  name "upscayl"
```
Comment (bevanjkay) [inline Casks/u/upscayl.rb:7]:
```suggestion
  name "Upscayl"
```

**Review by bevanjkay — CHANGES_REQUESTED** [review]

File: Casks/u/upscayl.rb (hunk 1 above)
Comment (aaronliu0130) [reply Casks/u/upscayl.rb:8]: Could I make it "AI image upscaler GUI for Real-ESRGAN-ncnn-vulkan", or is that too much details?

Comment (bevanjkay) [reply Casks/u/upscayl.rb:8]: I think my suggestion fits the `desc` guidelines better - https://docs.brew.sh/Cask-Cookbook#stanza-desc

**Review by bevanjkay — APPROVED** [review]

**Comment by aaronliu0130** [comment]
Thanks for that fix!

**Review by bevanjkay — APPROVED** [review]

**Pull request merged**